test-services/splunk/run-search.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+SFX_CMD=${SFX_CMD:-tosfx}
+QUERY='index=_internal group=per_index_thruput series!=_* | rename ev AS eventCount | rename kb AS kilobytes | table _time kilobytes eventCount series host | `gauge(kilobytes)` | `gauge(eventCount)`'
```

Comment:
Make a test helper in Python that can run an arbitrary query and then put the query used directly in the test code so that it is clear what is being tested.